baggage/HTTP_HEADER_FORMAT.md

```diff
@@ -77,8 +77,7 @@ If there are multiple `baggage` headers, all limits apply to the combination of
 
 1. A platform MUST propagate all `list-member`s up to _at least_ 64 `list-member`s including any `list-member`s added by the platform.
```

Comment:
```suggestion
1. A platform MUST propagate all `list-member`s including any `list-member`s added by the platform whenever both of these conditions are met:

**Condition 1:** The resulting `baggage-string` contains 64 `list-member`s or less.
**Condition 2:** The resulting `baggage-string` is of size 8192 bytes or less. 

2. If the resulting `baggage-string` would contain more than 64 `list-member`s, a platform MAY drop certain `list-member`s at its own discretion until the resulting `baggage-string` contains 64 `list-member`s or less.

3. If the resulting `baggage-string` would contain more than 8192 bytes, a platform MAY drop certain `list-member`s at its own discretion until the resulting `baggage-string` is of size 8192 bytes or less.
```

Reply:
I like this, but I would not separate 2 and 3, as it's a lot of nearly identical wording just to say that if C1 or C2 or both are violated then drop the items as you see fit.